qa.css

```diff
@@ -133,9 +139,3 @@ body:before {
   display: inline;
   font-weight: bold;
 }
-
-/* Update indicator */
-
-body {
-  background-image: url(http://hell.meiert.org/core/png/qa-1.4.png) !important;
```

Comment:
Why remove?

This is a key feature actually for it seems to be the most elegant way to inform about updates.

(That said, having the indicator on GitHub as opposed to hell.meiert.org is probably even more elegant :)